templates/zerver/help/zulip-on-windows.md

```diff
@@ -0,0 +1,44 @@
+# Using Zulip on Windows
+
+Zulip has a Windows client, compatible with Windows versions 7, 8 and 10 and is identical in
```

Comment:
"Zulip has a Windows client compatible with Windows versions 7, 8 and 10; it is identical in"